package.json

```diff
@@ -33,6 +44,9 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.14.2",
     "eslint-plugin-react-hooks": "^1.6.1",
+    "husky": "^3.0.3",
```

Comment:
One more trivial thing is, about commit message, you didn't `implement` `husky` and `lint-staged`; instead, you can say you `Introduce husky and lint-staged`. :P

Reply:
Oh, yes you are right. I can see the difference, thanks!